src/main/java/core/basesyntax/dao/StorageDao.java

```diff
@@ -6,4 +6,6 @@ public interface StorageDao {
     User add(User user);
 
     User get(String login);
+
```

Comment:
Consider changing the return type of the `get` method to `Optional<User>` to handle cases where a user might not be found, thus preventing potential `NullPointerException`.